tmva/tmva/inc/TMVA/Executor.h

```diff
@@ -46,9 +46,9 @@ class Executor {
    //////////////////////////////////////
    /// Default constructor of TMVA Executor class
    /// if ROOT::EnableImplicitMT has not been called then by default a serial executor will be created
-   /// A user can create a thread pool and enable multi-thread excution by calling TMVA::Config::Instance()::EnableMT(nthreads)
-   /// For releasing the thread pool used by TMVA one can do it by calling  TMVA::Config::Instance()::DisableMT() or
-   /// calling TMVA::Config::Instance()::EnableMT() with only one thread
+   /// A user can create a thread pool and enable multi-thread excution by calling TMVA::Config::Instance()::%EnableMT(nthreads)
+   /// For releasing the thread pool used by TMVA one can do it by calling  TMVA::Config::Instance()::%DisableMT() or
```

Comment:
We should format this as a C++ text:

/// A user can create a thread pool and enable multi-thread excution by calling
///                          TMVA::Config::Instance()::EnableMT(int nthreads);
///
/// For releasing the thread pool used by TMVA one can do it by calling 
///
///                  TMVA::Config::Instance()::DisableMT();

Reply:
The issue here is with:

```
/home/sftnight/build/workspace/root-makedoc-master/rootspi/rdoc/src/master/tmva/tmva/inc/TMVA/Executor.h:49: warning: explicit link request to 'EnableMT(nthreads)' could not be resolved
/home/sftnight/build/workspace/root-makedoc-master/rootspi/rdoc/src/master/tmva/tmva/inc/TMVA/Executor.h:50: warning: explicit link request to 'DisableMT()' could not be resolved
/home/sftnight/build/workspace/root-makedoc-master/rootspi/rdoc/src/master/tmva/tmva/inc/TMVA/Executor.h:51: warning: explicit link request to 'EnableMT()' could not be resolved
```

The % removes the link request and does not appear in the HTML output. I think reformatting it to C++ will not hide that warning, but not sure.

Reply:
formatted as markdown code now